docs/fourier-build-guide.md

```diff
@@ -0,0 +1,197 @@
+ # Fourier Build Guide
+
+## Parts List
+
+Here's a list of parts needed for the build:
+
+* 1 [Fourier PCB Kit](https://keeb.io/collections/split-keyboard-parts/products/fourier-40-split-staggered-keyboard)
+  * 2 Reset Switches \(included in kit\)
+  * 2 4.7kΩ resistors \(included in kit\)
+  * 2 TRRS Jacks \(included in kit\)
+  * 47 1N4148 diodes - through hole \(included in kit\) and [SMD diodes](https://keeb.io/products/1n4148-diodes) supported
+
+* 1 [Fourier Case Plates](https://keeb.io/collections/frontpage/products/fourier-keyboard-case-plates)
+  * 28 M2 6mm screws \(included with plates\)
+  * 14 M2 10mm standoffs \(included with plates\)
+  
+* 2 [Arduino Pro Micros](https://keeb.io/products/pro-micro-5v-16mhz-arduino-compatible-atmega32u4)
+* 1 TRRS Cable()
+* 1 Micro USB Cable()
+* PCB-mount MX stabilizers()
+* MX or Alps compatible keyswitches
+* [WS2812B Compatible RGB LED strip](https://keeb.io/collections/frontpage/products/rgb-led-strips-sk6812-ws2812b-compatible) \(optional, for underglow\)
+
+## Build Steps
+
+1. Prepare components
+2. Solder components
+    * Solder diodes
+    * Solder push button
+    * Solder 4.7kΩ resistors \(optional\)
+    * Solder Pro Micro header pins
+3. Solder switches
+4. Flash Pro Micro
+5. Solder Pro Micro
+6. Solder RGB strip \(optional\)
+
+## Prepare Components
+
+![](https://i.imgur.com/6YMFVgg.jpg)
+
+Grab your PCBs. Pictured bottoms up -- left is on the left, right is on the right.
+
+![](https://i.imgur.com/rrey3ej.jpg)
+
+
+If you're using through hole diodes, bend 'em up. Here, I'm just bending it around my finger
+
+![](https://i.imgur.com/sKo655O.jpg)
+
+
+Another way to do it, resistors shown here
+
+![]((https://i.imgur.com/2D39Ojx.jpg)
```

Comment:
Need to delete extra `(` here

Reply:
Deleted, also added the missing link for TRRS cable and stabilizers.